rllab/envs/mujoco/walker2d_env.py

```diff
@@ -55,3 +57,6 @@ def log_diagnostics(self, paths):
         logger.record_tabular('MaxForwardProgress', np.max(progs))
         logger.record_tabular('MinForwardProgress', np.min(progs))
         logger.record_tabular('StdForwardProgress', np.std(progs))
+
+    def log_diagnostics(self, paths):
```

Comment:
This should be covered by inheritance. Did you forget to call the super constructor?